src/igvfd/searches/configs/lab.py

```diff
@@ -21,7 +27,7 @@ def lab():
             'aliases': {
                 'title': 'Aliases'
             },
-            'awards': {
+            'awards.name': {
```

Comment:
It seemed more appropriate to me to use the award name instead of component specifically for labs since other objects might need the more functional component while labs would better tie to the alphanumeric name, was my figuring.